src/components/timeline/index.css

```diff
@@ -68,20 +68,19 @@
   height: 17px;
   padding: 0 7px 0 20px;
   margin: 4px 5px 0;
-  border: 0;
-  background-color: var(--blue-60);
+  border: none;
+  background-color: var(--grey-90-a10);
   border-radius: 12px;
-  color: #fff;
   font: inherit;
   line-height: 17px;
 }
 
 .timelineSettingsHiddenTracks:hover {
-  background-color: var(--blue-70);
+  background-color: var(--grey-90-a20);
 }
 
-.timelineSettingsHiddenTracks:active {
-  background-color: var(--blue-80);
+.timelineSettingsHiddenTracks:hover:active {
+  background-color: var(--grey-90-a30);
 }
```

Comment:
I reused the same colors as for the checkbox/radio buttons. But didn't use a border here because it felt a bit too "strong" and we have some text that should be enough for accessibility.